install-instructions.md

```diff
@@ -148,27 +155,9 @@ A simple installer for vanilla launcher [is planned](https://github.com/Fabulous
 
 ### GDLauncher
 
-Due to technical limits and issues, GDLauncher is no longer supported. It is recommended to migrate to Prism Launcher:
-
-1. [Install and run Prism Launcher](https://prismlauncher.org/)
-2. Click `Add Instance`
-3. Select `Modrinth` tab from the left
-4. Select "Fabulously Optimized"
-5. Click `OK`
-6. The modpack will now install.
-7. Once installed, click the `Folder` button on the left.
-8. Open GDLauncher
-9. Right click on your previously used instance → `Open Folder`
-10. Copy the important files and folders over:
-    * `saves` for your local worlds
-    * `resourcepacks`, if you use any (do not copy old versions of FO-bundled packs though)
-    * `screenshots`
-    * `servers.dat` for your multiplayer servers
-    * `options.txt`, if you want to keep your vanilla option changes
-11. Launch the instance on Prism and confirm everything is correct
-12. Uninstall GDLauncher
-
-[GDLauncher Carbon](https://gdlauncher.com/en/blog/curseforge-partnership-announcement/) may be supported in the future.
+Due to technical limits and issues, GDLauncher is no longer supported. It is recommended to migrate to the new GDLauncher Carbon.
+
+To be continued...
```

Comment:
todo

Reply:
Actually I think Carbon can already import all instances of the old one, so we could just note "These instructions refer to the new version of GDLauncher, codenamed Carbon. The old GDLauncher is not supported." below the new section

Reply:
This should be fixed within the latest commit (https://github.com/Fabulously-Optimized/wiki/pull/109/commits/7133e69756bd59b992c4d6ca1f8528e4cf9c9a6b). I have added instruction on how to migrate from the old GDLauncher to the new GDLauncher Carbon.